core/server/api/shared/validators/input/all.js

```diff
@@ -50,7 +50,8 @@ const validate = (config, attrs) => {
                 });
 
                 if (unallowedValues.length) {
-                    errors.push(new common.errors.ValidationError());
+                    const message = `Validation for "${key}" failed: ${unallowedValues.join(',')}`;
```

Comment:
There is no need to generate a message here, you can use the translation key.
There are a lot of examples how to use i18n 👍